docs/non_containerized_NSFS.md

```diff
@@ -264,6 +264,12 @@ node src/cmd/manage_nsfs bucket add --config_root ../standalon/config_root --fro
 ```
 NSFS management CLI command will create both account and bucket dir if it's missing in the config_root path.
 
+## NSFS Certificate
+
+Non containerized NSFS certificate location is configured in system.json file under the property `nsfs_ssl_cert_dir`. System will use a cert from this dir to create a valid HTTPS connection. If cert is missing in this dir a self-signed SSL certificate will be generated. Make sure the path mentioned in `nsfs_ssl_cert_dir` is valid before running nsfs command, If the path is invalid then cert flow will fail.
+
```

Comment:
maybe explain that we expect two files under the directory and that they have to be named tls.key and tls.crt.